doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,453 @@
+  * Occupancy sensing cluster with features like radar, vision, and ambient sensing.
+  * Intermittently Connected Devices feature with enhancements for the Long Idle Time (LIT) and Check-In protocol.
+    With these enhancements, the state of this feature is changed from provisional to certifiable.
+
+Thread
+------
+
+* Added Kconfig options for configuring the MLE child update timeout, child supervision interval, and child supervision check timeout.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.6.0 and platform v5.1.7 (``v3.11.6.0+5.1.7``).
+    They contain several fixes related to malfunctioning in a heavy traffic environment and more.
+    For details, see :ref:`zboss_changelog`.
+  * The :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.5.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Machine learning
+----------------
+
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+IPC radio firmware
+------------------
+
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system in all devices that contain MRAM, such as the nRF54H Series devices.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF Desktop
+-----------
+
+* Updated:
+
+  * The :ref:`nrf_desktop_settings_loader` to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology.
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations were migrated from the NVS settings backend to the ZMS settings backend.
+  * :ref:`nrf_desktop_watchdog` by adding the :ref:`zephyr:nrf54h20dk_nrf54h20` release configuration.
+  * Updated the configuration files of the :ref:`nrf_desktop_click_detector` (:file:`click_detector_def.h`) to allow them to be used even when Bluetooth LE peer control using a dedicated button (:ref:`CONFIG_DESKTOP_BLE_PEER_CONTROL <config_desktop_app_options>`) is disabled.
+  * The DTS description for board targets with a different DTS overlay file for each build type to isolate the common configuration that is now defined in the :file:`app_common.dtsi` file.
+    The :ref:`zephyr:nrf54h20dk_nrf54h20` board configuration has been updated.
+
+  * The :ref:`nrf_desktop_ble_conn_params` with the following changes:
```

Comment:
nRF54H20 DK does not use `nrf_desktop_ble_conn_params` module. No support for dongle (Bluetooth Central role)

Reply:
Will remove this